tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
   "extends": "@tsconfig/svelte/tsconfig.json",
   "include": ["src/components/**/*", "src/svelteDirectives/**/*"],
-  "exclude": ["node_modules/*", "__sapper__/*", "public/*"]
+  "exclude": ["node_modules/*", "__sapper__/*", "public/*"],
+  "compilerOptions": {
+    "strictNullChecks": false,
+    "noImplicitAny": false
```

Comment:
Just curious, why are we disabling `noImplicitAny`?

Reply:
Because the old `svelte-config` allowed implicit any, and the new one doesn't and I didn't want to convert all our code :laughing: 

As a bonus, I have a bit of a controversial opinion - I don't mind implicit any in the first place.

Reply:
> Because the old svelte-config allowed implicit any, and the new one doesn't and I didn't want to convert all our code 😆

Makes sense!